### PR TITLE
Nwood newtoc

### PR DESCRIPTION
**Pull request by NicholasWoodIMG**
Simplified ToC for the security model, as discussed in emails.

### CODE REVIEW

**Review by elliops — APPROVED** [review]
New simplified TOC as discussed in the group

File: specification/src/chapter1.adoc [hunk 1]
```diff
@@ -1,16 +1,72 @@
+
 [[chapter1]]
 
 == Introduction
 
-=== GLOSSARY/TAXONOMY
+This specifications provides guidelines for how Risc-V systems can use Risc-V security building blocks to build secure systems for different use cases.
```
Comment (rsahita) [inline specification/src/chapter1.adoc:6]:
```suggestion
This specifications provides guidelines for how RISC-V systems can use RISC-V security building blocks to build secure systems for different use cases.
```

File: specification/src/chapter1.adoc [hunk 2]
```diff
@@ -1,16 +1,72 @@
...
+
+It does this through a few example uses cases based on commonly used profiles. This is not intended to be exhaustive, it is expected that the principles described in the chosen example are general enough to be applicable to other use cases as well. But the examples may be extended over time as required.
```
Comment (rsahita) [inline specification/src/chapter1.adoc:8]:
```suggestion
The guidelines are listed through a few example uses cases based on commonly used profiles. The guidelines are not intended to be exhaustive; it is expected that the principles described in the chosen examples are general enough to be applicable to other use cases as well. The examples may be extended over time as required.
```

File: specification/src/chapter1.adoc [hunk 3]
```diff
@@ -1,16 +1,72 @@
...
+The requirements use the key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" that are to be interpreted as described in https://www.ietf.org/rfc/rfc2119.txt[RFC 2119] when, and only when, they appear in all capitals, as shown here. When these words are not capitalized, they have their normal English meanings.             
+|===
+
+A requirement or a group of requirements may be followed by non-normative text providing context or justification for the requirement. The non-normative text may also be used to reference sources that are the origin of the requirement.
+
+Trackable requirements are intended for ease of reference across dependant specifications. 
+
+=== Relationship to external profiles
+
+For the purpose of this specification, external profiles apply to existing ecosystems or segments, but do not generally mandate implementations or architectures. This specification does not aim to establish new profiles. Its main purpose is to provide guidelines for how Risc-V security building blocks can be used to build Risc-V products which can comply with existing profiles.
```
Comment (rsahita) [inline specification/src/chapter1.adoc:31]:
```suggestion
For the purpose of this specification, external profiles apply to existing ecosystems or segments, but do not generally mandate implementations or architectures. This specification does not aim to establish new profiles. Its main purpose is to provide guidelines for how RISC-V security building blocks can be used to build RISC-V products which can comply with existing profiles.
```

File: specification/src/chapter4.adoc [hunk 4]
```diff
@@ -1,3 +1,62 @@
...
+==== Isolation model
+
+==== Device access control
+
+==== Sealing
+
+==== Attestation
+
+
+=== Confidential compute system (Cove)
```
Comment (rsahita) [inline specification/src/chapter4.adoc:44]:
```suggestion
=== Confidential computing on RISC-V (CoVE)
```

File: specification/src/chapter3.adoc [hunk 5]
```diff
@@ -1,63 +1,29 @@
...
-//Entropy defenses (LWC, PQC)
-//Flushing defenses (fences)
-include::section_3_f.adoc[]
-
-//This section must cover the following security model description
-//Physical Adversary Attack Resistance [Paul Elliott]
-//Supply-chain Attack Resistance [Paul Elliott]
-//Discovery & Config Schema
-include::section_3_g.adoc[]
+== Risc-V security building blocks
```
Comment (rsahita) [inline specification/src/chapter3.adoc:3]:
```suggestion
== RISC-V security building blocks
```

File: specification/src/chapter3.adoc [hunk 6]
```diff
@@ -1,63 +1,29 @@
...
+
+=== Roadmap
+
+==== Capability based architecture
+
+Cheri/Capstone
+
+==== Memory safety
+
+==== Lightweight isolation
```
Comment (rsahita) [inline specification/src/chapter3.adoc:23]:
```suggestion
==== Lightweight isolation

==== Compartmentalization
```

File: specification/src/chapter4.adoc [hunk 7]
```diff
@@ -1,3 +1,62 @@
...
+==== Overview
+
+==== Isolation model
```
Comment (rsahita) [inline specification/src/chapter4.adoc:47]:
```suggestion
==== Isolation model

==== Physical threat protection
```

File: specification/src/chapter4.adoc [hunk 8]
```diff
@@ -1,3 +1,62 @@
...
+
+==== Device access control
+
+==== Trusted device assignment
+
+==== Sealing
+
+==== Attestation
```
Comment (rsahita) [inline specification/src/chapter4.adoc:55]:
```suggestion
==== Attestation

==== Debug, QoS and Performance Monitoring
```

File: specification/src/chapter3.adoc [hunk 9]
```diff
@@ -1,63 +1,29 @@
...
+
+==== System level isolation
+
+WorldGuard
+
+==== Cryptography enhancements
```
Comment (rsahita) [inline specification/src/chapter3.adoc:29]:
```suggestion
==== Cryptography enhancements

==== Cryptographic Data Protection 

=== Side-channel considerations [Luis Fiolhais]
Spatial timing channels and defense approaches
Temporal side channels and defense approaches

=== Supply chain considerations [Paul Elliott]

```

File: specification/src/chapter4.adoc [hunk 10]
```diff
@@ -1,3 +1,62 @@
 [[chapter4]]
 
-== Security Ecosystem
+== Use case examples
+
+=== Basic non-virtualized system
+
+==== Overview
```
Comment (rsahita) [inline specification/src/chapter4.adoc:7]:
```suggestion
==== Overview

==== Secure and Verified Boot
```

File: specification/src/chapter4.adoc [hunk 11]
```diff
@@ -1,3 +1,62 @@
...
+
+==== Device access control
+
+==== Sealing
+
+==== Attestation
+
+
+=== Confidential compute system (Cove)
+==== Overview
```
Comment (rsahita) [inline specification/src/chapter4.adoc:45]:
```suggestion
==== Overview

==== Measured Boot
```

File: specification/src/chapter3.adoc [hunk 12]
```diff
@@ -1,63 +1,29 @@
...
-//Flushing defenses (fences)
-include::section_3_f.adoc[]
-
-//This section must cover the following security model description
-//Physical Adversary Attack Resistance [Paul Elliott]
-//Supply-chain Attack Resistance [Paul Elliott]
-//Discovery & Config Schema
-include::section_3_g.adoc[]
+== Risc-V security building blocks
+
```
Comment (rsahita) [inline specification/src/chapter3.adoc:4]:
```suggestion
=== Root of trust
RTM for measurement, reporting and updates
Device Lifecycle management
Layered attestation
Authorized Debug

```

**Review by rsahita — APPROVED** [review]
I added some missing sections - overall LGTM - merging.

**Comment by rsahita** [comment]
Merging since Paul and I have reviewed the changes - note to the other authors - some placeholders have moved to different chapters.

**Pull request merged**